src/test/java/uk/gov/dhsc/htbhf/RunAcceptanceTests.java

```diff
@@ -6,7 +6,8 @@
 
 @RunWith(Cucumber.class)
 @CucumberOptions(plugin = "pretty",
-        features = "src/test/resources/features"
+        features = "src/test/resources/features",
+        strict = true
```

Comment:
We need the same config on RunCompatibilityTests too

Reply:
Yes, good spot, done.